ChangeBGColor/index.js

```diff
@@ -0,0 +1,2 @@
+let button = document.querySelector("input");
```

Comment:
Este `let` devia ser um `const`
```suggestion
const button = document.querySelector("input");
```